app/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Factorio Server Manager</title>
+    <link rel="icon"
+          type="image/png"
+          href="./dist/dist/img/favicon.ico"
```

Comment:
Thanks for the contribution!

Can we make this a single line and make sure there is a closing > tag.